deploy/penny-discord-bot-stack.yml

```diff
@@ -417,6 +421,58 @@ Resources:
       LogGroupName: /aws/lambda/GHHooksLambda-lambda
       RetentionInDays: 30
 
+  # GH-OAuth lambda config
+
+  ghOAuthIntegration:
+    Type: AWS::ApiGatewayV2::Integration
+    Properties:
+      Description: Lambda GHOAuth Integration
+      ApiId: !Ref apiGateway
+      ConnectionType: INTERNET
+      IntegrationMethod: POST
+      IntegrationType: AWS_PROXY
+      IntegrationUri: !GetAtt ghOAuthFunction.Arn
+      PayloadFormatVersion: '2.0'
+
+  ghOAuthRoutePost:
+    Type: AWS::ApiGatewayV2::Route
+    Properties:
+      ApiId: !Ref apiGateway
+      RouteKey: 'POST /gh-oauth'
+      Target: !Sub "integrations/${ghOAuthIntegration}"
+
+  ghOAuthFunction:
+    Type: AWS::Lambda::Function
+    Properties:
+      Code:
+        S3Bucket: penny-discord-bot
+        S3Key: !Sub ${GHOAuthFilename}.zip
+      Environment:
+        Variables:
+          BOT_TOKEN_ARN: arn:aws:secretsmanager:eu-west-1:177420307256:secret:prod/penny/penny-bot/discord-bot-token-VmhBXH
+          WH_SECRET_ARN: arn:aws:secretsmanager:eu-west-1:177420307256:secret:prod/penny/penny-bot/github-webhook-secret-MASfKL
+          LOG_LEVEL: trace
```

Comment:
You could keep log-level at trace for now till we are done debugging the lambda an we are sure it works OK.